xgcm/grid.py

```diff
@@ -799,6 +832,18 @@ def __init__(
             coordinates in ``ds``.
         metrics : dict, optional
             Specification of grid metrics
+        boundary : {None, 'fill', 'extend', 'extrapolate'}
```

Comment:
Should we be able to define different boundaries per axis like in `periodic`? E.g. `boundary={'X':'fill', 'Y':'extend'}`

Reply:
ah I see that is implemented below. I think its worth putting into the docstring though.